src/content/docs/dropins/checkout/tutorials/add-payment-method.mdx

```diff
@@ -9,131 +9,90 @@ import { Tabs, TabItem, Code, Steps } from '@astrojs/starlight/components';
+To render the Braintree payment method UI in the `PaymentMethods` container, you must define a modifier that you can use in the container's `main` slot.
 
-<Steps>
-1. Open the `blocks/commerce-checkout/commerce-checkout.js` file.
+```js
+const renderBraintreePayments = (element, context) => {
+  const { onPlaceOrder } = context;
+  const $content = document.createElement('div');
 
-1. Import the required files.
-   
-   ```js
-   ...
-   // Adyen drop-in modules
-   import * as adyenApi from '@dropins/adyen-checkout-extension/api.js';
-   import { render as AdyenProvider } from '@dropins/adyen-checkout-extension/render.js';
-   import AdyenPaymentMethod from '@dropins/adyen-checkout-extension/containers/AdyenPaymentMethod.js';
-   ...
-   ```
+  $content.innerHTML = '<div id="dropin-container"></div>';
+```
 
-1. Register the Adyen API.
+</Task>
 
-   ```js
-   ...
-   initializers.register(adyenApi.initialize, {});
-   ...
-   ```
+<Task>
 
-1. Use the `PaymentMethod` [handlers slot](/dropins/checkout/slots/) to add a payment method handler for Adyen. This handler renders the Adyen payment method UI when the backend indicates that Adyen credit card is an allowed payment method.
+### Create and insert the Braintree drop-in UI
 
-   ```js
-   ...
-   context.addPaymentMethodHandler('adyen_cc', {
-      render: (ctx, element) => {
-         AdyenProvider(AdyenPaymentMethod, ctx, element);
-         if (element) {
-            // Clear the lement first
-            element.innerHTML = '';
-            // Render the Adyen component
-            adyenProvider.render(AdyenPaymentMethod, ctx)(element);
-         }
-      },
-   })
-   ...
-   ```
-</Steps>
+Inside the function, a new `div` element is created and assigned to `$content`. This `div` contains another `div` with the ID `dropin-container`. If the element parameter is provided, its inner HTML is set to the content of `$content`. Then, the Braintree drop-in UI is created and attached to the `#dropin-container` using the provided authorization token.
+
+```js
+  if (element) {
+    element.innerHTML = $content.innerHTML;
+    braintree.dropin.create({
+      authorization: 'sandbox_g42y39zw_348pk9cgf3bgyw2b',
+      container: '#dropin-container',
+    });
+  }
+};
+```
 
 </Task>
 
 <Task>
 
-### Verify the integration
-
-<Steps>
-1. Start the development server to view the boilerplate storefront.
-
-   ```bash
-   npm start
-   ```
+### Set the function on the slot
 
-1. Navigate to a checkout page and verify that the Adyen payment method is displayed as an option.
+Finally, you must set the `renderBraintreePayments` function on the `main` slot.
 
-   [screenshot]
-
-</Steps>
+```js
+CheckoutProvider.render(PaymentMethods, {
+  slots: {
+    Main: renderBraintreePayments,
+  },
+})(document.getElementById('payment-methods'));
+```
```

Comment:
I usually appreciate when tutorials show the entire output combined at the end. Will we have this?

Reply:
That's good feedback. Yes, I think we should add that. I've done something similar in https://github.com/commerce-docs/microsite-commerce-storefront/pull/123, but I'm not sure if we need to show the whole block file in this particular context.